src/common/load-plugins.js

```diff
@@ -16,45 +36,68 @@ function loadPlugins(plugins) {
     require("../language-vue")
   ];
 
-  const externalPlugins = plugins
-    .concat(
-      getPluginsFromPackage(
-        readPkgUp.sync({
-          normalize: false
-        }).pkg
-      )
-    )
-    .map(plugin => {
-      if (typeof plugin !== "string") {
-        return plugin;
-      }
-
-      const pluginPath = resolve.sync(plugin, { basedir: process.cwd() });
-      return Object.assign({ name: plugin }, eval("require")(pluginPath));
+  const externalPluginInfos = [];
```

Comment:
It'd be better to not mutate array if possible as it's clearer, I'd prefer something like this:

```js
const externalManualLoadPluginInfos = plugins.map(pluginName => ({
  name: pluginName,
  requirePath: resolve.sync(pluginName, { basedir: process.cwd() })
}));

const externalAutoLoadPluginInfos = pluginSearchDirs
  .map(pluginSearchDir => {
    const resolvedPluginSearchDir = path.resolve(
      process.cwd(),
      pluginSearchDir
    );

    if (!isDirectory(pluginSearchDir)) {
      throw new Error(
        `${pluginSearchDir} does not exist or is not a directory`
      );
    }

    const nodeModulesDir = path.resolve(
      resolvedPluginSearchDir,
      "node_modules"
    );

    return findPluginsInNodeModules(nodeModulesDir).map(pluginName => ({
      name: pluginName,
      requirePath: resolve.sync(pluginName, {
        basedir: resolvedPluginSearchDir
      })
    }));
  })
  .reduce((a, b) => a.concat(b), []);

const externalPlugins = uniqBy(
  externalManualLoadPluginInfos.concat(externalAutoLoadPluginInfos),
  "requirePath"
).map(externalPluginInfo =>
  Object.assign(
    { name: externalPluginInfo.name },
    eval("require")(externalPluginInfo.requirePath)
  )
);
```